### PR TITLE
Image Upload functionality

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone I;
- [x] Implemented the layout of the image upload container to blend with the project design.
- [x] Updated the logout button in the sidebar to make it visible to all mobile screen sizes.
- [x] Styled the image upload container.
Thanks for reviewing 💯

### CODE REVIEW

File: src/components/addCostumer/AddCostumer.css [hunk 1]
```diff
@@ -19,11 +19,28 @@
...
   margin: 20px;
+  text-align: center;
+}
+
+.p-text {
+  color: rgb(26, 24, 24);
+  margin: 0 0 10px -20px;
+  text-align: center;
+  font-weight: bold;
 }
```
Comment (brytebee) [inline src/components/addCostumer/AddCostumer.css:45]: - [ ] **Required:** Awesome job so far. However, consider making the edit customer match the design you have the edit user.

Edit Customer | Edit User
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Anny85-code 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Your positioning is brilliant.✔️
- The colours you have chosen match the existing design.✔️
- consistency in design.✔️
- No linter or console errors.✔️
- The app functions correctly.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

File: src/components/addCostumer/AddCostumer.css (hunk 1 above)
Comment (Anny85-code) [reply src/components/addCostumer/AddCostumer.css:45]: Thanks for your review, changes implemented.

**Review by brytebee — APPROVED** [review]
Hi, @Anny85-code 👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉

**Pull request merged**